docs/source/model_design/vivarium_model_components/risk_factors/risk_mediation/index.rst

```diff
@@ -44,214 +45,504 @@ but that effect is (completely) mediated by another risk factor R2.
-  therefore GBD never models such mediation situations.
-
-This "mediation factor" can be used to calculate values for each of the causal effects represented by arrows
-in the diagram in one of two ways; the following sections describe how to do this.
-
-Unfortunately, GBD mediation factors are inconsistent with the multiplicative application of RRs.
-This is because mediation factors are defined as proportions of the *excess* risk -- the amount the RR is
-greater than 1.
-In effect, GBD's mediation methods can split e.g. an RR of 2.2 into an RR of 1.7 and an RR of 1.6 (since 0.7 + 0.6 = 1.2, which
-is the overall RR's excess risk).
-However, if we apply these two RRs, we end up with an RR of of 1.7 * 1.6 = 2.72, which is higher than
-we started with.
-For this reason, we have to decide whether to be consistent with how we apply RRs
-(and therefore stay calibrated to overall prevalence),
-or to be consistent with GBD's mediation methodology.
-In past models, we have mostly done the latter; we recommend doing the former in future models.
-Both approaches are described below.
+Framing
++++++++
 
-.. todo::
+GBD never models effects between causes, but it does model effects of risk factors
+on causes, and some effects of risk factors on other risk factors.
+These last two kinds of effects create the possibility of mediation in GBD:
+when one risk factor affects another, which affects a cause in turn.
+GBD calls the upstream risk factor the "distal risk."
 
-  We don't totally understand why GBD models *excess* risk -- while there is a possibility that
-  there is a good reason behind it, we haven't been able to find that reason, so we prefer
-  Approach 1 for now.
+.. image:: gbd-mediation.drawio.svg
 
-If GBD does not report a mediation factor, it may still report one or more of the effects in the diagram.
-The rest can be filled in with external sources.
+GBD may model any number of mediators between a single risk factor-cause pair,
+in addition to a direct effect.
 
 .. note::
-  Be careful not to interpret GBD RRs as *direct* RRs -- unless there is a mediation factor,
-  the GBD RR will not be adjusted for any mediators.
 
-All our multiplicative math is based on this equation describing how RRs are applied:
+  GBD currently models every risk factor-mediator-cause triple separately,
+  which means its results can imply multiple conflicting values for the same effect in
+  this diagram.
+  This may have been addressed for GBD 2021.
 
-.. math::
-
-  RR_\text{r→o} = RR_\text{r→o,direct} * \prod_{m \in M} (RR_\text{m→o})^{\Delta_\text{r→m}}
+.. todo::
 
-where :math:`M` is the set of mediators that mediate the relationship between risk :math:`r`
-and outcome :math:`o` and :math:`\Delta_\text{r→m}` is the (assumed linear) increase in :math:`m`
-caused by a one-unit increase in :math:`r`.
+  We have yet to make a simulation that uses a combination of risks where GBD implies
+  multi-layer mediation.
```

Comment:
Well I guess we did in CVD but we didn't really consider it in depth and don't have a recommended approach here.

Reply:
Oops -- did not know this when I wrote this sentence. I'll tweak it a bit.